lib/build-view.js

```diff
@@ -182,22 +185,35 @@ module.exports = (function() {
     this.titleText.text('Aborted!');
   };
 
+  BuildView.prototype._render = function() {
+    var string = this.a2h.toHtml(_.escape(this.buffer.toString('utf8')));
+    _.forEach(this.links, function (link, index) {
+      string = string.replace(link.text, '<a id="' + link.id + '">' + _.escape(link.text) + '</a>');
+    });
```

Comment:
When I tested this, I found two bugs:
 * you must replace `_.escape(link.text)` instead of `link.text`, because `string` is escaped and `link.text` is not
 * the replace fails, when there are two `link.text`, which are the same (`<a>` is wrapped around the first match twice and not around the second)